app/controllers/balance_transactions_controller.rb

```diff
@@ -0,0 +1,73 @@
+class BalanceTransactionsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @balance_transaction = BalanceTransaction.new
+    @balance_transaction.transactions.build
+  end
+
+  def create
+    byebug
+    respond_to do |format|
+      notice = BalanceTransactions::Creator.new(create_params).call ? t('.success') : t('.failure')
```

Comment:
@sinpai I think, that you can move `BalanceTransactions::Creator.new(create_params)` to method, like 

``` ruby
def create
   byebug
    respond_to do |format|
      notice = balance_transaction_creator.call ? t('.success') : t('.failure')
      format.html { redirect_to transactions_path, notice: notice }
      format.js { render layout: false }
    end
end

def balance_transaction_creator
  BalanceTransactions::Creator.new(create_params)
end